cli/server/previewServer.js

```diff
@@ -30,6 +30,26 @@ const PUBLIC_CONFIG_BASENAME = 'cache-public.config';
 const PUBLIC_CONFIG_PATH = path.join(CACHE_DIRECTORY, PUBLIC_CONFIG_BASENAME);
 const FAV_ICON_PATH = './node_modules/jest-preview/cli/server/favicon.ico';
 
+const updateNotifier = require('update-notifier');
+const chalk = require('chalk'); // Already installed in `update-notifier`
+
+// Checks for available update
+const notifier = updateNotifier({
+  pkg: require('../../package.json'),
+  updateCheckInterval: 0, // How often to check for updates
+  shouldNotifyInNpmScript: true, // Allows notification to be shown when running as an npm script
+  distTag: 'latest' // Can be use to notify user about pre-relase version
+})
+
+notifier.notify({
+  defer: true, // Try not to annoy user by showing the notification after the process has exited
```

Comment:
@huynhducduy Can you elaborate on `defer` property? I'm not sure I understand how this notifier can annoy users when the process has exited. Thanks.

Reply:
@nvh95 If `defer` is set to `true`, the notification only displays when the user exits `jest-preview`'s process (by Ctrl+C, sending interrupts,...). This is very handy when the user runs the server concurently together with jest watcher (For example: `run-p jest-preview 'test -- --watch --no-cache'`. Because jest will clear the terminal's output right away, and the notification will appear and then disappear immediately (flashing).

Reply:
It makes sense. Let's keep it that way!